pkgs/by-name/hc/hclust2/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, nix-update-script
+, python3Packages
+, python3
+, phylophlan
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "hclust2";
+  version = "1.0.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "SegataLab";
+    repo = "hclust2";
+    rev = version;
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/${version}";
```